docker/Dockerfile.base

```diff
@@ -0,0 +1,8 @@
+FROM phusion/baseimage:focal-1.0.0
+
+# Copy over all Palace build files for this image
+COPY ./docker /ls_build
```

Comment:
After this layer, you should be able to do a lot of the setup that doesn't require the source code, so that we have some layer caching.

Reply:
This will require splitting the work done in `simplified_app.sh` into two scripts, one that does setup not requiring source code, and one that requires the application source code. That way we can copy the docker setup, do the general OS setup (installing packages etc), then have a second script that actually sets up the app using our source code.